### ISSUE TITLE
Video  - No Automatic Play 

### ISSUE THREAD

**Issue by jinky32** [ThreeUK/digital-library.three.co.uk#16]
**Requirement**:  audio or video with audio should not play automatically on a web page.

**Why is this required?** If audio plays automatically any screen reader user will be unable to hear the output of their screen reader.

**Comment by beasers** [comment]
Does this apply to brand-led interstitial takeover pages? ;-)

**Comment by VinceButcher** [comment]
All pages
